src/fondue/Fondue.js

```diff
@@ -188,28 +188,33 @@ export default class Fondue {
 	// Usage:
 	//   fondue.features
 	get features() {
+		const featureResult = {};
 		const result = this._raw("GSUB");
 		const featuresRaw = result;
-		const features = result.featureList.featureRecords.map(
-			(record) => record.featureTag
-		);
-		const scripts = result.scriptList.scriptRecords.map(
-			(record) => record.scriptTag
-		);
-		// features = features.filter((feature, index) => features.indexOf(feature) === index); // remove doubles
-		const featureResult = {};
-		features.forEach((feature, index) => {
-			if (!featureResult[feature]) {
-				featureResult[feature] = {
-					...layoutFeature[feature],
-					scripts: {},
+
+		if (result) {
+			const features = result.featureList.featureRecords.map(
+				(record) => record.featureTag
+			);
+			const scripts = result.scriptList.scriptRecords.map(
+				(record) => record.scriptTag
+			);
+			// features = features.filter((feature, index) => features.indexOf(feature) === index); // remove doubles
+			features.forEach((feature, index) => {
```

Comment:
I see this code is mostly shuffled around but I don't really understand what this code is doing? Perhaps it could be restructured a bit to make it clearer, or maybe I'm just missing some domain knowledge here?

Reply:
Same as before: original code, just using it as-is for now. Will explore and refactor when I dive deeper into this specific bit. But it indeed looks it can be simplified a lot.